config/map_fetcher.go

```diff
@@ -0,0 +1,28 @@
+package config
+
+import "strconv"
+
+// mapFetcher provides an implementation of the Fetcher interface by wrapping
+// the `map[string]string` type.
+type mapFetcher map[string]string
```

Comment:
It's important to point out that mapFetcher is not safe from concurrent map access and must therefore be protected by other mutexes.